test/apps/iodemo/io_demo.cc

```diff
@@ -2607,6 +2613,11 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
             std::cout << "  -D                          Enable debugging mode for IO operation timeouts" << std::endl;
             std::cout << "  -H                          Use human-readable timestamps" << std::endl;
             std::cout << "  -P <interval>               Set report printing interval"  << std::endl;
+            std::cout << "  -R <rndv-thresh>            Rendezvous threshold used to force eager" << std::endl;
+            std::cout << "                              or rendezvous protocol:" << std::endl;
+            std::cout << "                                - if not set - threshold is selected by UCX" << std::endl;
+            std::cout << "                                - to force always use eager protocol - set value larger" << std::endl;
+            std::cout << "                                  than max data can be sent (see -d option)" << std::endl;
```

Comment:
```
- R <threshold>   Always use rendezvous protocol for messages larger than 
                  this size, and eager protocol for messages lower than this size.
                  If not set, the threshold is selected automatically by UCX

Reply:
updated

Reply:
then what protocol will be used for messages of exactly the `size`? Maybe
`Always use rendezvous protocol for messages starting from this size..`?

Reply:
no problemo, re-worded